pkg/cli/settings.go

```diff
@@ -95,22 +93,35 @@ func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.RepositoryCache, "repository-cache", s.RepositoryCache, "path to the file containing cached repository indexes")
 }
 
+// GetNamespace gets the namespace from the configuration
+func (s *Settings) GetNamespace() string {
```

Comment:
In Go it's considered bad form to use `GetNamespace` as a function name. Instead, it should be `Namespace`. This comes from https://golang.org/doc/effective_go.html#Getters (in the Go documentation) and there are linting tools that remind us not to name things that way.

This is a Go convention that's a little different from other languages.